graphrag/index/workflows/factory.py

```diff
@@ -66,4 +83,5 @@ def _get_workflows_list(
                 "create_final_text_units",
                 "create_community_reports_text",
                 "generate_text_embeddings",
+                *(update_workflows if is_update_run else []),
```

Comment:
FGR does not do covariates, so you may want to test if this is ok

Reply:
I see below you have a guard, so this could probably be ignored

Reply:
the covariates workflow will just do nothing if there's no covariates :)